backend/src/vantracking/cache.py

```diff
@@ -0,0 +1,54 @@
+from abc import ABC, abstractmethod
+
+from src.model.stop import Stop
+from src.vantracking.location import Location
+
+
+class VanStateCache(ABC):
```

Comment:
Both of the implementations look fine, but I'm a little confused fundamentally as to why this is necessary?

We only have one production environment and hence only one will be used, so why did we implement two? (sorry if I missed this discussion in Discord, I've been ill)

Reply:
Basically, the memcached implementation (Which @tbwrigh should have done) can be trivially switched into in the chance that we need to scale horizontally.

Reply:
Will implement memcached at some point not a required feature at the moment.